admin/views/paper-collapsible.php

```diff
@@ -36,7 +36,7 @@ class="<?php echo esc_attr( 'yoast-paper ' . $class ); ?>"<?php echo ( $paper_id
 			}
 
 			printf(
-				'<h2 class="%1$s"><button%2$s type="button" class="toggleable-container-trigger" aria-expanded="%3$s">%4$s%5$s <span class="toggleable-container-icon dashicons %6$s" aria-hidden="true"></span></button></h2>',
+				'<h2 class="%1$s"><div%2$s type="button" class="toggleable-container-trigger" aria-expanded="%3$s">%4$s%5$s <span class="toggleable-container-icon dashicons %6$s" aria-hidden="true"></span></div></h2>',
```

Comment:
This is incorrect, for many reasons. It should stay a `<button>` element.
We're not going to introduce clickable divs 🙂